src/style.css

```diff
@@ -47,12 +49,26 @@ body {
   /* outline: red solid 2px; */
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: